lib/animina/accounts/resources/basic_user.ex

```diff
@@ -91,8 +107,9 @@ defmodule Animina.Accounts.BasicUser do
   end
 
   identities do
-    identity :unique_email, [:email]
-    identity :unique_username, [:username]
+    identity :unique_email, [:email], eager_check_with: Animina.Accounts
```

Comment:
Wouldn't it improve code readability if we'd use an alias for `Animina.Accounts`?